### PR TITLE
add note about proto. add makecar soln.

### PR DESCRIPTION
**Pull request by amaseda**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Learning Objectives
 
 - Explain the importance of OOJS
-- Describe the role of ES2015 Classes and how they work
+- Describe the role of ES6 Classes and how they work
```
Comment (nayana487) [inline readme.md]: why did you change to ES6? ES2015 is the accurate, official name. https://bytearcher.com/articles/es6-vs-es2015-name/

File: readme.md [hunk 2]
```diff
@@ -351,6 +376,17 @@ class Dog extends Animal {
 
 -------
 
+### Further Reading: Class vs. Prototypical Inheritance
+
+When reading about object oriented programming, you'll come across two popular paradigms: **class** vs. **prototypical**. Examples of class-based programming languages are Ruby and Java. Even though we are using something called a "class" in Javascript, however, Javascript employs prototypical inheritance.
+
+Classes became a part of Javascript with the implementation of ES6. Under the hood, however, classes are using constructor functions and prototypes, two fundamental tools that were used to implement OOJS prior to ES6.
+
+We will not be explicitly covering prototypical inheritance in this course. If you are interested in learning more, however, check out these resources...
```
Comment (nayana487) [inline readme.md]: so under the hood, what's happening with ES2015 is still prototypal inheritance so i dunno if this is what we wanna say here

**Comment by nayana487** [comment]
apparently can't leave a comment where you didn't make changes, but lines 63 & 66 -- there's "front-end" and "back end". pick one: https://dzone.com/articles/blogging-programmers-style

File: readme.md [hunk 3]
```diff
@@ -117,6 +117,31 @@ Define a function `makeCar` that takes two parameters - `model` and `color` - an
...
+    return {
+      model: model,
+      color: color
+    }
+  }
+  ```
+
+</details>
+
 This is the basic idea behind OOP: we define a blueprint for an object and use it to generate multiple instances of it!
```
Comment (nayana487) [inline readme.md:143]: maybe note that instance is a technical term and it IS an object. whereas a class is NOT an object, but merely a blueprint.

**Comment by nayana487** [comment]
this line: `Unlike object notation, you do not need to use commas when separating class methods.`, before the ATM exercise. another moment to hammer in that an ES2015 class is not an object, just a blueprint

**Comment by nayana487** [comment]
i had an article i read that i can't find now that put this better, but in case you're asked why we don't put methods in constructors, it's because the point of the constructor is to create an instance with initial properties. technically, it will work, but does hurt in processing and also, generally doesn't make sense to be there.

File: readme.md (hunk 1 above)
Comment (amaseda) [reply readme.md]: True. Changing back.

File: readme.md (hunk 2 above)
Comment (amaseda) [reply readme.md]: Point here is to clarify that this is not true classical inheritance and that actually prototypical inheritance is what's happening under the hood.

I'll reword it. Didn't come out right the first time.

File: readme.md (hunk 3 above)
Comment (amaseda) [reply readme.md:143]: :+1:

**Comment by amaseda** [comment]
@nayana487 Thanks for the suggestions. If you end up finding that article about methods in constructors let me know.
